unnamed_toaster/unnamed_toaster/scripts/shooter_math.py

```diff
@@ -38,19 +38,21 @@
 from std_msgs.msg import Float64
 
 
-class AutoShooter:
+class ShooterMath:
 
     def __init__(self):
-        self.pub = rospy.Publisher("turret_command", Float64, queue_size=1)
+        #self.pub = rospy.Publisher("turret_command", Float64, queue_size=1)
 
         # Setup TF2
         # http://wiki.ros.org/tf2/Tutorials/Writing%20a%20tf2%20listener%20%28Python%29
         self.tf_buffer = tf2_ros.Buffer()
         self.tf_listener = tf2_ros.TransformListener(self.tf_buffer)
```

Comment:
rather than creating two buffers and listeners in this node - I'd suggest you pass the buffer into the __init__() of this class from the outer MatchLogic instance.

Reply:
Im not sure i know how to do that :/